prowler/providers/azure/services/defender/defender_auto_provisioning_log_analytics_agent_vms_on/defender_auto_provisioning_log_analytics_agent_vms_on.py

```diff
@@ -0,0 +1,23 @@
+from prowler.lib.check.models import Check, Check_Report_Azure
+from prowler.providers.azure.services.defender.defender_client import defender_client
+
+
+class defender_auto_provisioning_log_analytics_agent_vms_on(Check):
+    def execute(self) -> Check_Report_Azure:
+        findings = []
+
+        for subscription_name, auto_provisioning_settings in defender_client.auto_provisioning_settings.items():
+            report = Check_Report_Azure(self.metadata())
+            report.status = "PASS"
+            report.subscription = subscription_name
+            report.resource_name = auto_provisioning_settings["default"].resource_name
```

Comment:
I'm still not sure if we should include here a condition just to review the `default` resource. Let's talk with @sergargar on Monday.